automatic/flashplayeractivex/update.ps1

```diff
@@ -1,7 +1,9 @@
-
+﻿
 import-module au
+. "$PSScriptRoot\..\..\scripts\Get-Padded-Version.ps1"
 
 $releases = "https://get.adobe.com/en/flashplayer/" # URL to for GetLatest
+$padVersionUnder = '24.0.1'
```

Comment:
I can of course remove this, which would mean that every version that is a 4-part version would be padded with zeros.